app/pages/cif/project-revision/[projectRevision]/view.tsx

```diff
@@ -111,7 +223,10 @@ export function ProjectRevisionView({
             tagName={"dl"}
             className="project-revision-view-form"
             schema={
-              createProjectRevisionViewSchema(allRevisionTypes) as JSONSchema7
+              createProjectRevisionViewSchema(
+                allRevisionTypes,
+                filteredRevisionStatuses
+              ) as JSONSchema7
```

Comment:
casting here could be avoided if we declare the return type of `createProjectRevisionViewSchema(..)`